lms/djangoapps/verify_student/tests/test_views.py

```diff
@@ -1853,3 +1879,168 @@ def test_403_for_non_staff(self):
         url = reverse('verification_photo_urls', kwargs={'receipt_id': six.text_type(self.receipt_id)})
         response = self.client.get(url)
         self.assertEqual(response.status_code, 403)
+
+
+@patch.dict(settings.FEATURES, {
```

Comment:
We no longer need to patch this setting, right?

Reply:
I just refactored the tests so that this setting doesn't need to be patched